interop-testing/src/main/java/io/grpc/testing/integration/AbstractInteropTest.java

```diff
@@ -270,15 +281,15 @@ public ClientStreamTracer newClientStreamTracer(CallOptions callOptions, Metadat
    */
   @Before
   public void setUp() {
+    clientStatsRecorder = new FakeStatsRecorder();
```

Comment:
Initialize the stats recorders at their declaration (like before)? It doesn't seem like this needs to be delayed until `@Before`

Reply:
Done.